packages/flutter/lib/src/widgets/image.dart

```diff
@@ -464,7 +464,10 @@ class Image extends StatefulWidget {
     this.filterQuality = FilterQuality.low,
     int? cacheWidth,
     int? cacheHeight,
-  }) : image = ResizeImage.resizeIfNeeded(cacheWidth, cacheHeight, FileImage(file, scale: scale)),
+  }) :
+       // FileImage is not supported on Flutter Web therefore neither this method.
+       assert(!kIsWeb, 'Image.file is not supported on Flutter Web.'),
```

Comment:
The error message could offer `Image.asset` and `Image.network` as alternatives.

Reply:
Great catch. Done. PTAL